src/fheroes2/system/settings.h

```diff
@@ -354,6 +360,21 @@ class Settings
         _viewWorldZoomLevel = zoomLevel;
     }
 
+    SaveFileSortingMethod GetSaveFileSortingMethod() const
```

Comment:
Please start this method's name from the lower character:
```suggestion
    SaveFileSortingMethod getSaveFileSortingMethod() const
```